ensembl/conf/ini-files/meleagris_gallopavo.ini

```
@@ -70,13 +70,8 @@ ENSEMBL_SEARCH_IDXS     = [ Domain Family Gene GenomicAlignment Marker Sequence
 
 
 
-VARIATION_PARAM   = rs119118426
-VARIATION_TEXT    = rs119118426
-
-PHENOTYPE_PARAM   = 2
-PHENOTYPE_TEXT    = Feather colour, bronze
-
-SEARCH_TEXT       = cancer 
+VARIATION_PARAM   = rs119111991;vf=12:4639131:C_A:EVA
+VARIATION_TEXT    = rs119111991
```

same, keep the search text - 

```suggestion
VARIATION_TEXT    = rs119111991

SEARCH_TEXT = cancer
```